src/tree/ResolvedWebAppResource.ts

```diff
@@ -21,9 +21,16 @@ import { NotAvailableTreeItem } from './NotAvailableTreeItem';
 import { SiteTreeItem } from './SiteTreeItem';
 import { WebJobsNATreeItem, WebJobsTreeItem } from './WebJobsTreeItem';
 
+export function isResolvedWebAppResource(ti: unknown): ti is ResolvedWebAppResource {
+    return (ti as unknown as ResolvedWebAppResource).kind === ResolvedWebAppResource.kind;
```

Comment:
Why check `kind`? I think that for App Service, the `kind` is always going to be blank so this will probably always be true.  If we're just making sure it's a `SiteTreeItem`, we should be able to check if the context matches with `azApp`

Reply:
Or we could check `type`?

Reply:
Pardon the naming, but this is a _new_ `kind`.

https://github.com/microsoft/vscode-azureappservice/pull/2236/files#diff-33dbbf31b6cdf6477373c8a791073499ee49d90cd7abbd5cc44d4e2ad4c6af4fR31-R32

Reply:
Not sure if you saw my previous comment (because I did an edit) so bump.

Reply:
Got any ideas for a better idea?